lib/gooddata/bricks/middleware/logger_middleware.rb

```diff
@@ -14,14 +12,31 @@ class LoggerMiddleware < Bricks::Middleware
       def call(params)
         params = params.to_hash
         logger = nil
+        collect_stats = false
+        collect_stats = true if params['COLLECT_STATS'] && params['COLLECT_STATS'].to_b
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)